service/src/test/java/uk/gov/hmcts/reform/fpl/utils/JudgeAndLegalAdvisorHelperTest.java

```diff
@@ -35,6 +35,18 @@ void shouldReturnEmptyJudgeNameWhenJudgeTitleIsNotProvided() {
         assertThat(judgeTitleAndName).isEqualTo("");
     }
 
+    @Test
+    void shouldReturnBlankPlaceholderWhenNoJudgeEnteredForDraftSDO() {
+        JudgeAndLegalAdvisor judgeAndLegalAdvisor = JudgeAndLegalAdvisor.builder()
+            .judgeTitle(null)
+            .build();
+
+        String judgeTitleAndName = JudgeAndLegalAdvisorHelper.formatJudgeTitleAndNameForDraftSDO(judgeAndLegalAdvisor,
```

Comment:
(0) what do you think about static import? Would it increase readability?

Reply:
agree 👍